src/common/mfu_flist.c

```diff
@@ -615,10 +639,16 @@ void mfu_flist_array_free(int levels, mfu_flist** outlists)
     return;
 }
 
-/* return number of files across all procs */
+/* return number of files or oids across all procs */
 uint64_t mfu_flist_global_size(mfu_flist bflist)
 {
     flist_t* flist = (flist_t*) bflist;
+#ifdef DAOS_SUPPORT
+    uint64_t total_oids = flist->total_oids;
+    if (total_oids > 0) {
+        return total_oids;
+    }
```

Comment:
Is the intent for this to fall through and return flist->total_files if oids == 0?

Reply:
@adammoody yes, but now that I think about it that doesn't necessary mean we should return total_files if that is the case. I could init total_oids to -1, and check for that instead?

Reply:
@adammoody since now that I think about it someone could be trying to perform a copy where there are no oids (or its empty).

Reply:
@adammoody well I actually think this is ok. Since if someone was actually trying to perform a posix copy it will fall through and return total_files. And if total_oids is zero, but the intent was to do a cont copy, then no copy will be performed (since they are both zero).

Reply:
@dsikich That's what I was thinking as well, though the concept of `total_items` would ultimately be simpler.